### PR TITLE
Get unread totals for all the teams a user is on.

### PR DESCRIPTION
**Pull request by andreistanciu24**
GET /users/{user_id}/teams/unread

### CODE REVIEW

File: source/v4/teams.yaml [hunk 1]
```diff
@@ -271,3 +271,37 @@
           $ref: '#/responses/Forbidden'
         '404':
           $ref: '#/responses/NotFound'
+
+  /users/{user_id}/teams/unread
+    get:
+      tags:
+        - teams
+      summary: Get the count for unread messages and mentions in the teams the user is a member of
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can we shorten this to `Get team unreads for a user`? It's better to keep the summaries short since they'll appear in a sidebar

File: source/v4/teams.yaml [hunk 2]
```diff
@@ -271,3 +271,37 @@
...
+          required: true
+          type: string
+        - name: exclude_team
+          in: query
+          description: Optional team id to be excluded from the results
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Teams found
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can we change this to `Team unreads retrieval successful`?

**Comment by andreistanciu24** [comment]
done
